### PR TITLE
connect with price update api

### PR DESCRIPTION
**Pull request by wangwh03**
API integration for price display and price update.

### CODE REVIEW

**Comment by wangwh03** [comment]
@rykann This is my last PR today. :) Will record a gif once it's a go on your side as well.

File: app/src/main/java/com/ghomebyrw/gworker/clients/TooltimeAPI.java [hunk 1]
```diff
@@ -35,5 +36,5 @@
     // TODO - update once API is ready
     @Headers("X-Auth-Token: admin-token")
     @POST("fieldworkers/login")
-    Call<Boolean> logIn(@Body LoginInfo loginInfo);
+    Call<Job> logIn(@Body LoginInfo loginInfo);
```
Comment (rykann) [inline app/src/main/java/com/ghomebyrw/gworker/clients/TooltimeAPI.java]: login returns a job?

**Comment by rykann** [comment]
I took a quick look. I have one question, but otherwise looks good to merge.

File: app/src/main/java/com/ghomebyrw/gworker/clients/TooltimeAPI.java (hunk 1 above)
Comment (wangwh03) [inline app/src/main/java/com/ghomebyrw/gworker/clients/TooltimeAPI.java]: This was changed by mistake reverting now. Thanks for catching this!

Comment (wangwh03) [inline app/src/main/java/com/ghomebyrw/gworker/clients/TooltimeAPI.java]: Reverted.

**Pull request merged**